setup.py

```diff
@@ -8,6 +8,7 @@
       author='Neal Fultz',
       author_email='nfultz@gmail.com',
       license='BSD',
+      python_requires='>=3.8, <4',
```

Comment:
Wouldn't that restrict the ability of someone to install the latest/upcoming version of nb2mail with python 2? I thought you wanted to still support python 2.

Reply:
would it be easier to rename to `nb2mail3` and leave the current package for legacy users?

Reply:
I am no a package expert.

I thought that if we will have two versions:
* current is 0.6
* new one would be 1.0 (or something else, you decide)

That way on python2 env, pip will automatically will choose 0.6 and on python3 env pip will pick 1.0 version

What do you guys think?